### PR TITLE
Canonizes Robot Yawns

### LINKED ISSUE #77592: Tired bots get included in the yawn chain
Reporting client version: 514.1589

## Round ID:
[212360](https://scrubby.melonmesa.com/round/212360)

## Testmerges:
- [Feature: bitrunner, a new supply role](https://www.github.com/tgstation/tgstation/pull/77259)

## Reproduction:
image
i genuinely think this is one of those basically non-issues that's mostly a "look, that's funny", but if someone would consider non-sentient silicons as not something that should yawn when prompted by others, then here's the issue for it

### PR DESCRIPTION
**Pull request by san7890**
## About The Pull Request

Closes #77592

I agree that it's silly that robots shouldn't be able to yawn, but I think that instead of removing the potential for this interaction to occur outright (which can be done with moving around a check), it should be embraced instead.

## Why It's Good For The Game

image

Robots fake-yawning to make them feel more realistic to the workers of the station (and if they aren't station based, their once-creators (which is fine because we removed swarmers)) and not being exempt from a really cutesy niche interaction greatly appeals to me.

## Changelog

:cl:
add: Ever see a robot yawn? Don't worry, it makes sense now.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
if you have a better idea for what the emote message should be let me know

**Comment by iwishforducks** [comment]
you should make them *buzz or something along those lines, the reasoning being that yawning is the robotic equivalent of dial-up internet. actually can we have robots have their own version of *yawn'ing that causes all other robots around them to also *dialup (or something like that)? if not i still think this is a cute interaction

**Comment by Jacquerel** [comment]
> you should make them *buzz or something along those lines, the reasoning being that yawning is the robotic equivalent of dial-up internet. actually can we have robots have their own version of *yawn'ing that causes all other robots around them to also *dialup (or something like that)? if not i still think this is a cute interaction

lets give the contagious dialup emote an ear-ripping dialup screech sound too what could go wrong
I like this PR though

**Review by Ghommie — APPROVED** [review]

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -531,6 +531,8 @@
 	key_third_person = "yawns"
 	message = "yawns."
 	message_mime = "acts out an exaggerated silent yawn."
+	message_robot = "symphathetically yawns."
```
Comment (Iamgoofball) [inline code/modules/mob/living/emote.dm:534]: suggestion
```
	message_robot = "engages their sympathy-yawn processor."
```

**Pull request merged**